.travis.yml

```diff
@@ -4,6 +4,11 @@ sudo: required
 services:
   - docker
 
+go:
+  - 1.6
+  - 1.7
+  - tip
```

Comment:
yes but it isn't actually doing much right now since the go version is hard coded within the dockerfile